.eslintrc

```diff
@@ -4,6 +4,9 @@
     "node": true,
     "mocha": true
   },
+  "parserOptions": {
+    "ecmaVersion": 6
```

Comment:
@rimiti  what is the difference between using this parser option and adding `es6: true` to the `env` attribute?